test/Controller/PromotionalCollectionControllerTest.php

```diff
@@ -0,0 +1,527 @@
+                'http://api.elifesciences.org/promotional-collections/1',
+                ['Accept' => 'application/vnd.elife.promotional-collection+json; version=1']
+            ),
+            new Response(
+                200,
+                ['Content-Type' => 'application/vnd.elife.promotional-collection+json; version=1'],
+                json_encode([
+                    'id' => '1',
+                    'title' => 'Promotional collection title',
+                    'published' => '2010-01-01T00:00:00Z',
+                    'updated' => '2011-01-01T00:00:00Z',
+                    'image' => [
+                        'banner' => [
+                            'uri' => 'https://www.example.com/iiif/ban%2Fner',
+                            'alt' => '',
+                            'source' => [
+                                'mediaType' => 'image/jpeg',
+                                'uri' => 'https://www.example.com/banner.jpg',
+                                'filename' => 'image.jpg',
+                            ],
+                            'size' => [
+                                'width' => 3872,
+                                'height' => 2581,
+                            ],
+                        ],
+                        'thumbnail' => [
+                            'uri' => 'https://www.example.com/iiif/image',
+                            'alt' => '',
+                            'source' => [
+                                'mediaType' => 'image/jpeg',
+                                'uri' => 'https://www.example.com/image.jpg',
+                                'filename' => 'image.jpg',
+                            ],
+                            'size' => [
+                                'width' => 800,
+                                'height' => 600,
+                            ],
+                        ],
+                    ],
+                    'impactStatement' => 'Promotional collection impact statement',
+                    'editors' => [
+                        [
+                            'id' => 'person',
+                            'type' => [
+                                'id' => 'senior-editor',
+                                'label' => 'Senior editor',
+                            ],
+                            'name' => [
+                                'preferred' => 'Person',
+                                'index' => 'Person',
+                            ],
+                        ],
+                    ],
+                    'content' => [
+                        [
+                            'type' => 'blog-article',
+                            'id' => '1',
+                            'title' => 'Blog article title',
+                            'published' => '2010-01-01T00:00:00Z',
+                        ],
+                    ],
+                ])
+            )
+        );
+
+        return '/highlights/1/promotional-collection-title';
```

Comment:
how do people get to this? Only direct linking seems to be provided.

Reply:
> how do people get to this? Only direct linking seems to be provided.

These are essentially landed pages that are collections. The will be included in marketing campaigns but there is no current requirement to allow people to access these through the Journal navigation.

Reply:
Thanks for the clarification. The next step is, how does the person creating the campaign (e.g. composing an email) get hold of the URL?

Reply:
> Thanks for the clarification. The next step is, how does the person creating the campaign (e.g. composing an email) get hold of the URL?

Good point, this can be handled in journal-cms. This functionality exists in Journal CMS for other content but it hasn't been implemented yet in the promotional collections PR.

Reply:
This has been added to the Journal CMS PR: https://github.com/elifesciences/journal-cms/pull/501/commits/9018463b4819c1e38f29043866304db608b7412a